services/web/server/tests/unit/isolated/test_diagnostics_healthcheck.py

```diff
@@ -77,6 +78,7 @@ def mock_environment(mock_env_devel_environment: Dict[str, str], monkeypatch):
     monkeypatch.setenv("DIAGNOSTICS_MAX_TASK_DELAY", f"{SLOW_HANDLER_DELAY_SECS}")
     monkeypatch.setenv("DIAGNOSTICS_MAX_AVG_LATENCY", f"{2.0}")
     monkeypatch.setenv("DIAGNOSTICS_START_SENSING_DELAY", f"{0}")  # inmidiately
+    monkeypatch.setenv("SC_HEALTHCHECK_TIMEOUT", "2m")
```

Comment:
again: I'd prefer to unit handling in timings

Reply:
Please read https://docs.docker.com/engine/reference/builder/#healthcheck